src/ui/RedditListCell.tsx

```diff
@@ -1,14 +1,14 @@
-import {Composite, ImageView, TextView} from 'tabris';
+import {Composite, ImageView, TextView, Properties} from 'tabris';
 import {component, property} from 'tabris-decorators';
 import {RedditPostData} from '../common';
 
 @component export default class RedditListCell extends Composite {
```

Comment:
This should extend Cell instead of Composite, that way we have less nesting in the ListView. Same for the ReditGalleryCell